### PR TITLE
add for rh-demo

### PR DESCRIPTION
**Pull request by sallyom**
- added a new Containerfile with rh-python
- added a var for MODEL_FILE to customize which one to use

### CODE REVIEW

File: chat-rh.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+class Chat:
+
+    n_ctx = 2048
+
+    def __init__(self) -> None:
+        self.chat_history = [
+                {"role": "system", "content": """You are a helpful assistant that is comfortable speaking
+                 with C level executives in a professional setting."""},
+                ]
+        self.llm = Llama(model_path="/locallm/nested/dir/llama-2-7b-chat.Q4_K_M.gguf",
```
Comment (sallyom) [inline chat-rh.py]: that #L12 is the only line that is different from chat.py

Comment (MichaelClifford) [reply chat-rh.py]: Might be easier to just add `model_path` parameter to the the chat class `__init__`, instead of managing another file.

Comment (sallyom) [reply chat-rh.py]: yup was thinking that but also was being lazy

**Review by MichaelClifford — APPROVED** [review]
LGTM! Thanks for the PR @sallyom 🎉

**Pull request merged**